dll/win32/shell32/CSendToMenu.cpp

```diff
@@ -295,6 +295,11 @@ CSendToMenu::QueryContextMenu(HMENU hMenu,
     TRACE("%p %p %u %u %u %u\n", this,
           hMenu, indexMenu, idCmdFirst, idCmdLast, uFlags);
 
+    if ((uFlags & (CMF_NOVERBS | CMF_VERBSONLY)) != 0) // CORE-16544
```

Comment:
If you believe your PR to be a proper solution I would recommend to strip the mentioning of CORE-16544 here and at the other places where you added that.
We do use such comments in the code only for temporary hacks and workarounds. (If you believe your PR would fall into that category, then it would be okay to keep them, but then the PRs title/description should be changed to reflect it being a hack.)
For now I do assume your PR is a proper fix. (?!?)